src/components/header/header.js

```diff
@@ -232,7 +236,7 @@ const Header = (props) => {
                           >
                             <img src={IconCompare} alt="Compare Icon" />
                             <span className="badge bg-success rounded-circle">
-                              {wishlistCount}
+                              3
```

Comment:
@Prutha-Upadhyay, You've hard coded this 3 here, removing `{wishlistCount}` Count you fix this ?